web/themes/joinup/joinup_theme.theme

```diff
@@ -19,13 +19,77 @@ use Drupal\file\Entity\File;
 use Drupal\image\Entity\ImageStyle;
 
 /**
+ * Check if route is special.
+ *
+ * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
+ *   The route object.
+ *
+ * @return array
+ *   The route type.
+ */
+function joinup_check_route(RouteMatchInterface $route_match) {
+  $route_name = $route_match->getRouteName();
+
+  // Declare available types.
+  $types = [
+    'create' => FALSE,
+    'tabs' => FALSE,
+  ];
+
+  // Check if page has a create form.
+  $create_content_routes = [
+    'collection.propose_form',
+    'rdf_entity.propose_form',
+  ];
+
+  // Check if string has 'add' on the end.
+  if (preg_match('/.add$/', $route_name)) {
+    $types['create'] = TRUE;
+  }
+
+  if (!empty($route_name) && in_array($route_name, $create_content_routes)) {
+    $types['create'] = TRUE;
+  }
+
+  // Check if page has form vertical tabs.
+  $tabs_routes = [
+    'entity.rdf_entity.edit_form',
+    'rdf_entity.propose_form',
```

Comment:
`rdf_entity.propose_form` repeats in both sets. As we spoke today, if there are 2 types of forms, maybe there should be 2 separate functions, or an exclusion logic rather than inclusion. At any case, it's a repetition ..